scio-examples/src/main/scala/com/spotify/scio/examples/extra/MagnolifyAvroExample.scala

```diff
@@ -78,8 +69,8 @@ object MagnolifyAvroReadExample {
     import MagnolifyAvroExample._
 
     val (sc, args) = ContextAndArgs(cmdlineArgs)
-    sc.avroFile(args("input"), wordCountType.schema)
-      .map(e => wordCountType(e))
+    // uses implicitly-derived magnolify.avro.AvroType[WordCount] to read from avro
```

Comment:
Don't we require the `import magnolify.avro._` somewhere ?

Reply:
It is not required so long as the `AvroType[T]` is derivable via the built-ins in magnolify.